_layouts/ndt-test.html

```diff
@@ -2,33 +2,120 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta charset="utf8">
-	<title>{{ page.title }}</title>
-	<style>
-		#ndt-iframe {
-			width: 720px;
-			height: 480px;
-			border: none;
-		}
-	</style>
-</head>
-<body>
-<script>
-var	mlabNsUrl = 'https://mlab-ns.appspot.com/ndt?format=json';
-var xhr = new XMLHttpRequest();
-xhr.onreadystatechange = function () {
-  if (xhr.readyState === 4) {
-    if (xhr.status === 200) {
-      var response = JSON.parse(xhr.responseText);
-      var ndtIframe = document.createElement('iframe');
-      ndtIframe.src = response.url;
-      ndtIframe.setAttribute('id', 'ndt-iframe');
-      document.body.appendChild(ndtIframe);
+    <meta charset="utf8">
+    <title>{{ page.title }}</title>
+    <link href="{{ site.baseurl }}/css/ndt-test.css" rel="stylesheet">
+    <!--[if IE]>
+    <style>
+    @font-face {
+        font-family: "League Gothic";
+        src: url("../fonts/league-gothic/league_gothic.eot");
     }
-  }
-};
-xhr.open("GET", mlabNsUrl, true);
-xhr.send();
-</script>
+    </style>
+    <![endif]-->
+    <script>
+        // The gauge.min.js in the ndt submodule wants the font digital-7-mono,
+        // and by default looks for it in the /fonts directory.  Setting this
+        // variable tells it where to look.
+        var CANV_GAUGE_FONTS_PATH = "../fonts/digital-7-mono";
+    </script>
+    <script src="{{ site.baseurl }}/ndt/HTML5-frontend/jquery-1.12.1.min.js"></script>
+    <script src="{{ site.baseurl }}/ndt/HTML5-frontend/gauge.min.js"></script>
+    <script src="{{ site.baseurl }}/ndt/HTML5-frontend/script.js"></script>
+    <script src="{{ site.baseurl }}/ndt/HTML5-frontend/ndt-browser-client.js"></script>
+    <script src="{{ site.baseurl }}/ndt/HTML5-frontend/ndt-wrapper.js"></script>
+    <script>
+        // Look up a suitable NDT server using mlab-ns. The global variable
+        // ndtServer is used by by the backend javascript to determine which
+        // server to test against.
+        var ndtServer;
+        setMlabServer();
+        function setMlabServer() {
+            var mlabNsService = 'https:' == location.protocol ? 'ndt_ssl' : 'ndt';
+            var mlabNsUrl = 'https://mlab-ns.appspot.com/';
+            $.ajax({
+                url: mlabNsUrl + mlabNsService + '?format=json',
+                dataType: 'json',
+                async: false,
```

Comment:
> To avoid this is why I've set "async: false" in the $.ajax() request. My understanding is that this should block the entire page until the call finishes. 

Ah, can we add a comment above this line saying it's deliberately false so that the page doesn't load until we get the result from mlab-ns?